packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -241,72 +238,58 @@ function FiatExchangeOptions({ route, navigation }: Props) {
           {t(isCashIn ? 'selectPaymentMethod' : 'selectCashOutMethod')}
         </Text>
         <View style={styles.paymentMethodsContainer}>
-          {isCashIn && (
-            <>
-              <PaymentMethodRadioItem
-                text={t('payWithCard')}
-                selected={selectedPaymentMethod === PaymentMethod.CARD}
-                onSelect={onSelectPaymentMethod(PaymentMethod.CARD)}
-              />
-              <PaymentMethodRadioItem
-                text={t('payWithBank')}
-                selected={selectedPaymentMethod === PaymentMethod.BANK}
-                onSelect={onSelectPaymentMethod(PaymentMethod.BANK)}
-              />
-            </>
-          )}
-          <PaymentMethodRadioItem
-            text={t('payWithExchange')}
-            selected={selectedPaymentMethod === PaymentMethod.EXCHANGE}
-            onSelect={onSelectPaymentMethod(PaymentMethod.EXCHANGE)}
-          />
-          {!isCashIn && (
-            <>
-              <PaymentMethodRadioItem
-                text={t('receiveOnAddress')}
-                selected={selectedPaymentMethod === PaymentMethod.ADDRESS}
-                onSelect={onSelectPaymentMethod(PaymentMethod.ADDRESS)}
-                enabled={selectedCurrency === CURRENCY_ENUM.GOLD}
-              />
+          {asyncLocalCicoProviders.loading ? (
```

Comment:
Would it be worth to handle the error state?

Reply:
I don't think so but can be convinced otherwise. My reasoning is that the only time there should be an error is if there is an issue with the Firebase connection, which we already show an error for